driftbase/resources/friends.py

```diff
@@ -0,0 +1,47 @@
+"""
+Friendships configuration
```

Comment:
New resources need to be added to config/config.json

Reply:
Or they won't get auto-registered when provisioned

Reply:
It's already in the config.

Reply:
Right I missed it